experimental/dds/tree2/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -1075,10 +1075,10 @@ describe("SequenceField - Compose", () => {
 			[0, 1, 2],
 			[2, 1, 0],
 		]) {
-			const move1 = tagChange(Change.move(a, 1, b), tag1);
-			const move2 = tagChange(Change.move(b, 1, c), tag2);
+			const move1 = tagChange(Change.move(a, 1, b > a ? b + 1 : b), tag1);
+			const move2 = tagChange(Change.move(b, 1, c > b ? c + 1 : c), tag2);
```

Comment:
It might be nice to make a change builder flavor which interprets indices in the old way.

Reply:
I'd be more inclined to add that if we weren't going to remove those builders in the first place.